pyramid/scripts/pserve.py

```diff
@@ -334,6 +341,17 @@ def serve():
                     msg = ''
                 self.out('Exiting%s (-v to see traceback)' % msg)
 
+        if self.options.browser:
+            def open_browser():
+                context = loadcontext(SERVER, app_spec, name=app_name, relative_to=base,
+                        global_conf=vars)
+                url = 'http://{host}:{port}/'.format(**context.config())
```

Comment:
shouldn't this always be localhost? the "host" option is specific to an individual wsgi server and is usually the interface to listen on, not the ip address to access. Technically the port is also specific to the wsgi server but probably acceptable.